modules/administration-guide/partials/proc_configuring-number-of-replicas.adoc

```diff
@@ -0,0 +1,68 @@
+:_content-type: PROCEDURE
+:description: Configuring a number of replicas
+:keywords: administration-guide, configuring-number-of-replicas
+:navtitle: Configuring a number of replicas {prod-cli}
+:page-aliases:
+
+[id="configuring-number-of-replicas"]
+= Configuring a number of replicas
+
+To configure the number of replicas for {prod-short} operands using {kubernetes} `HorizontalPodAutoscaler`
+(HPA), you can define an `HPA` resource for deployment. The `HPA` dynamically adjusts the number
+of replicas based on specified metrics.
+
+.Procedure
+
+. Create an `HPA` resource for a deployment, specifying the target metrics and desired replica count.
++
+[source,yaml,subs="+attributes,+quotes"]
+----
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: scaler
+  namespace: {prod-namespace}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: __<deployment_name>__ <1>
+  ...
+----
+<1> The `__<deployment_name>__` corresponds to the one following deployments:
+* `{prod-deployment}`
+* `che-gateway`
+* `{prod-deployment}-dashboard`
+* `plugin-registry`
+* `devfile-registry`
+
+.Create a `HorizontalPodAutoscaler` for {prod-deployment} deployment:
```

Comment:
```suggestion
. Create a `HorizontalPodAutoscaler` for {prod-deployment} deployment:
```

Reply:
It is a example title. ` ` is not allowed.

Reply:
I thought it's the next step in the instructions. sorry, my bad.